request.go

```diff
@@ -18,9 +19,25 @@ func (r *Request) Get(url string) (*http.Response, error) {
 		return nil, err
 	}
 
+	if len(r.headers) > 0 {
+		for key, val := range r.headers {
+			req.Header.Add(key, val)
+		}
+	}
+
 	return r.client.doRequest(req)
 }
 
+func (r *Request) SetHeader(h, v string) *Request {
+	r.headers[h] = v
+
+	return r
+}
```

Comment:
As headers can have more than one value, I think it's interesting to add a method to add more value to the header like [http.Header](https://pkg.go.dev/net/http#Header.Add)

Reply:
done changed for http.Header type